e2e/integration_tests/data_flow_test.go

```diff
@@ -139,40 +144,74 @@ func Test_EndpointLinksAreAvailable(t *testing.T) {
 	var id_npi_2 string
 	err = store.DB.QueryRow("SELECT id FROM npi_organizations WHERE npi_id='2222222222';").Scan(&id_npi_2)
 	failOnError(err)
+	var id_npi_3 string
```

Comment:
consider restructuring parts of this test to make it more readable and reduce repetition.
Have dicts for each of the objects you're checking:
(pseudocode):
```
ep1 := {
    url: <endpoint url>,
    org_name: <org name>,
    mapped_org_npi_ids: [<npi_id_1>,<npi_id_2>,...],
    ... other... - maybe expected value for tests if they are beyond "check that such and such is in the db"
}
ep2 := ...

ep_list := [ep1, ep2, ...]

for ep in ep_list:
   ... tests ...
}
```

This will collect the information together so there aren't ids like `npi_id='1588667794` without associations to anything, and make it so you only have to write the testing code once.